src/status_im/contexts/wallet/collectible/utils.cljs

```diff
@@ -2,14 +2,15 @@
   (:require [status-im.config :as config]
             [status-im.constants :as constants]
             [status-im.contexts.wallet.common.utils.networks :as network-utils]
-            [taoensso.timbre :as log]))
+            [taoensso.timbre :as log]
+            [utils.number :as utils.number]))
 
 (defn collectible-balance
   ([{:keys [ownership]} address]
-   (let [balance (some #(when (= address (:address %))
-                          (js/parseInt (:balance %)))
-                       ownership)]
-     (if (js/Number.isNaN balance) 0 balance))))
+   (->> ownership
+        (some #(when (= address (:address %))
+                 (:balance %)))
+        utils.number/parse-int)))
```

Comment:
Small refactoring of collectible balance util method based on https://github.com/status-im/status-mobile/pull/21871#discussion_r1900847647